modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/editor/BaseEditor.js

```diff
@@ -40,6 +63,49 @@ const BaseEditor = forwardRef(
 			});
 		}, []);
 
+		useEffect(() => {
+			if (!initialConfig.editorTransformerURLs) {
+				return;
+			}
+
+			loadClientExtensions([
+				{
+					clientExtensionDefinitions: initialConfig.editorTransformerURLs.map(
+						(url) => ({
+							importDeclaration: `default from ${url}`,
+						})
+					),
+					onLoad: (bindingContexts) => {
+						let transformedConfig = initialConfig;
+
+						bindingContexts.forEach(
+							({binding: editorTransformer, error}) => {
+								if (
+									process.env.NODE_ENV === 'development' &&
+									error
+								) {
+									console.error(error);
+								}
+
+								const editorConfigTransformer =
+									editorTransformer?.editorConfigTransformer;
+
+								if (editorConfigTransformer) {
+									transformedConfig = editorConfigTransformer(
+										transformedConfig
+									);
+								}
+							}
```

Comment:
I still feel that failing gracefully is preferred behavior for CKEditor CET. But now, the behavior is not fixed in util, but chosen in usage. Alternatively, we could show an error toast or some other error indicator in UI. That's the kinds of behavior that would make more sense to me if CET is implementing some critical part of the component. If I'm wrong, we can easily change this later on.

Reply:
yes, I agree. To me a graceful fail if EditorConfig CET can't load/run is to use the identity function as config transformer, which results in using the `config` given by the back-end.

A more in-depth approach would be to make `loadClientExtensions()` aware of the desired load semantics, so rather than [expecting all of them to load](https://github.com/liferay/liferay-portal/blob/345939f3f0d0cf729995be915cd85d5735ce783a/modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/utils/loadClientExtensions.ts#L45), be able to configure `loadClientExtensions()` to load as much as possible, reporting errors on individual binding contexts. I don't think we need to go that far at this moment, just providing this as a possible evolution.